src/model/user/user.ts

```diff
@@ -0,0 +1,18 @@
+export interface IUser
+{
+    _id: string;
+    firstName: string;
+    lastName: string;
+    email: string;
+    hashedPassword: string;
+    bio: string;
+    recipes: [
+        {
+            recipeId: string, 
+            Date: Date
+        }
+    ]
+}

```

Comment:
We'll need to type out the timestamp as well mongoose defaults it as `createdAt` and `updatedAt`
```suggestion

createdAt: Date;
updatedAt: Date;
}
```